cli/exec.go

```diff
@@ -77,6 +77,10 @@ func (e *ExecCommand) runExec(args []string) error {
 		Env:          e.Envs,
 	}
 
+	if err := checkTty(createExecConfig.AttachStdin, createExecConfig.Tty, os.Stdout.Fd()); err != nil {
```

Comment:
I think it should be os.Stdin.Fd() here.

Reply:
updated.